projects/1.21/assets/evilcraft/evilcraft/lang/zh_cn.json

```
@@ -0,0 +1,764 @@
+  "block.evilcraft.environmental_accumulator.info": "只能在暗黑神庙里被找到，无法制造。",
+  "block.evilcraft.undead_leaves": "亡灵树叶",
+  "block.evilcraft.undead_log": "亡灵原木",
+  "block.evilcraft.undead_log_stripped": "去皮亡灵原木",
+  "block.evilcraft.undead_wood": "亡灵木",
+  "block.evilcraft.undead_wood_stripped": "去皮亡灵木",
+  "block.evilcraft.undead_slab": "亡灵木台阶",
+  "block.evilcraft.undead_fence": "亡灵木栅栏",
+  "block.evilcraft.undead_fence_gate": "亡灵木栅栏门",
+  "block.evilcraft.undead_sapling": "亡灵树苗",
+  "block.evilcraft.undead_sapling.info": "长成一棵亡灵树。",
+  "block.evilcraft.undead_planks": "亡灵木板",
+  "block.evilcraft.invisible_redstone_block": "隐形红石块",
+  "block.evilcraft.purifier": "净化器",
+  "block.evilcraft.purifier.info": "祛魔台",
+  "block.evilcraft.dark_brick": "暗黑砖",
+  "block.evilcraft.dark_blood_brick": "暗黑血砖",
+  "block.evilcraft.spirit_furnace": "灵魂炉",
+  "block.evilcraft.spirit_furnace.info": "烹饪复仇之魂以获得其掉落物。",
+  "block.evilcraft.spirit_furnace.help.invalid": "无法开始烹饪",
+  "block.evilcraft.spirit_furnace.help.invalid.no_entity": "缺少灵魂箱/灵魂箱为空",
+  "block.evilcraft.spirit_furnace.help.invalid.content_size": "容量大小：%s",
+  "block.evilcraft.spirit_furnace.help.invalid.required_size": "需要大小：%s",
+  "block.evilcraft.spirit_furnace.help.invalid.force_halt": "储存空间不足",
+  "block.evilcraft.spirit_furnace.help.invalid.caught_error": "无效生物灵魂。请联系模组开发者。",
+  "block.evilcraft.box_of_eternal_closure": "永恒灵魂箱",
+  "block.evilcraft.box_of_eternal_closure.info.content": "存有：",
+  "block.evilcraft.dark_tank": "暗黑流体罐",
+  "block.evilcraft.dark_tank.info": "与其他流体罐合成以增加容量。Shift右击以切换自动补给。",
+  "block.evilcraft.dark_tank.info.auto_supply": "自动补给：%s",
+  "block.evilcraft.sanguinary_pedestal_0": "血腥基座",
+  "block.evilcraft.sanguinary_pedestal_0.info": "从附近的血迹中提取血液。",
+  "block.evilcraft.sanguinary_pedestal_1": "强化血腥基座",
+  "block.evilcraft.sanguinary_pedestal_1.info": "从附近的血迹中提取更多血液。",
+  "block.evilcraft.spiked_plate": "钉刺板",
+  "block.evilcraft.spiked_plate.info": "对生物造成伤害，放置在基座上以提取血液。",
+  "block.evilcraft.spirit_reanimator": "灵魂再生炉",
+  "block.evilcraft.spirit_reanimator.info": "复活复仇之魂。",
+  "block.evilcraft.spirit_reanimator.help.invalid": "无法再生",
+  "block.evilcraft.spirit_reanimator.help.invalid.no_entity": "缺少灵魂箱、灵魂箱无效或为空",
+  "block.evilcraft.spirit_reanimator.help.invalid.invalid_entity": "生物没有可用的刷怪蛋",
+  "block.evilcraft.spirit_reanimator.help.invalid.different_egg": "移除种类不同的刷怪蛋",
+  "block.evilcraft.entangled_chalice": "纠缠圣杯",
+  "block.evilcraft.entangled_chalice.info": "同组纠缠圣杯将共享流体存储。\\nShift右击以切换自动补给。",
+  "block.evilcraft.entangled_chalice.info.auto_supply": "自动补给：%s",
+  "block.evilcraft.entangled_chalice.info.id": "组：%s",
+  "block.evilcraft.dark_power_gem_block": "暗黑充能宝石块",
+  "block.evilcraft.gem_stone_torch": "宝石火把",
+  "block.evilcraft.gem_stone_torch.info": "阻止复仇之魂在15格半径内生成。",
+  "block.evilcraft.eternal_water": "永恒水方块",
+  "block.evilcraft.eternal_water.info": "一个无限的水源。",
+  "block.evilcraft.blood_waxed_coal_block": "血蜡煤块",
```

```suggestion
  "block.evilcraft.blood_waxed_coal_block": "涂血煤块",
```
Blood-Waxed和蜡一点关系没有，下同

---

暂改成`覆血煤`

---

那注血得了……

---

waxed这个操作不符合注血吧

---

在血能注入器中完成。
择你叫着顺口一版更改或不改。

---

暂定`覆血`吧